src/Trixi.jl

```diff
@@ -307,7 +307,7 @@ export PositivityPreservingLimiterZhangShu, EntropyBoundedLimiter
 export trixi_include, examples_dir, get_examples, default_example,
        default_example_unstructured, ode_default_options
 
-export ode_norm, ode_unstable_check
+export ode_norm, ode_unstable_check, unstable_check
```

Comment:
If we export it, it becomes part of our public API (although it is not documented...). However, I do not think this is a good idea right now since it can be easily confused with `ode_unstable_check`. I see two options right now:

- Do something specific for your subcell limiter ecosystem
- Make it general so that it works nicely with the other parts

Reply:
Okay, that's true.
I'm completely fine with the first option. But how would that look like in your opinion? Rename it to something like `unstable_check_SSPRK`, have an optional parameter `unstable_check = nothing` for `solve()` of the custom time integrators and add `unstable_check = unstable_check_SSRK` to every elixir, where it's needed?
Or really removing the parameter `unstable_check` from the solve function and hardcode it into the SSPRK method?